CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Thanks for contributing to Scala CLI!
+
+This doc is meant as a guide on how best to contribute to Scala CLI.
+
+## Creating issues
+
+Whenever you happen upon something that needs improvement, be sure to come back to us and create an issue. Please make
+use of the available templates and answer all the included questions, so that the maintenance team can understand your
+problem easier.
+
+## Pull requests
+
+### Fork-Pull
+
+We accept external pull requests according to
+the [standard GitHub fork-pull flow](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+Create a fork of this repository, commit your changes there and create a pull request. We try to review those as often
+as possible.
+
+### Rules for a well-formed PR
```

Comment:
Add rules when someone create PR with changes in documentation. In this case developers should open PR to `stable` branch.

Reply:
added a separate section on the `main` and `stable` branches, please verify if that's sufficient